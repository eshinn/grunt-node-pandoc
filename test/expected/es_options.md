# Section 1

Hello, section 1.

## Section 2

Hello section 2.
