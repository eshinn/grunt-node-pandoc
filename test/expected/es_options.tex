\section{Section 1}\label{section-1}

Hello, section 1.

\subsection{Section 2}\label{section-2}

Hello section 2.
